src/statusBar.ts

```diff
@@ -51,6 +51,28 @@ class StatusBarImpl implements vscode.Disposable {
     this._wasRecordingMacro = isRecordingMacro;
   }
 
+  public ReportLinesChanged(numLinesChanged: number, mode: ModeName) {
```

Comment:
Can you move these to another class/file? I'd like `statusBar` without having to maintain any state or logic of what the status bar should say.

Reply:
Not sure where the best spot of this would be though.

Reply:
I poked around neovim to get some inspiration, misc.1.c...so any opposition for me throwing it in util.ts?

Reply:
I also dont need the functions, they were just incase I wanted to clean up the prints a little/handle the non plural cases